### ISSUE TITLE
Rambox stays open even after it closes.

### ISSUE THREAD

**Issue by sujay1844** [ramboxapp/community-edition#2833]
When I close the rambox window, it just doesn't close. It's difficult to explain. It remains open in the wallpaper but clicking and scrolling doesn't work. It's like the window is embedded into my desktop. (Unfortunately I can't share screenshots due to privacy concerns.)

### ENVIRONMENT

Version: 0.7.6
Platform: linux (x64)
Electron: 7.2.4
Chromium: 78.0.3904.130
Node: 12.8.1

$ inxi -Fxz
System:
  Kernel: 5.8.6-1-MANJARO x86_64 bits: 64 compiler: N/A 
  Desktop: KDE Plasma 5.19.5 Distro: Manjaro Linux 
Machine:
  Type: Laptop System: Acer product: Swift SF314-42 v: V1.03 
  serial: <filter> 
  Mobo: RO model: Kona_RN v: V1.03 serial: <filter> UEFI: Insyde v: 1.03 
  date: 04/07/2020 
Battery:
  ID-1: BAT1 charge: 31.9 Wh condition: 42.6/47.9 Wh (89%) 
  model: Murata AP18C4K status: Discharging 
CPU:
  Topology: 6-Core model: AMD Ryzen 5 4500U with Radeon Graphics bits: 64 
  type: MCP arch: Zen rev: 1 L2 cache: 3072 KiB 
  flags: avx avx2 lm nx pae sse sse2 sse3 sse4_1 sse4_2 sse4a ssse3 svm 
  bogomips: 28453 
  Speed: 1406 MHz min/max: 1400/2375 MHz Core speeds (MHz): 1: 1397 2: 1397 
  3: 1397 4: 1397 5: 1397 6: 1397 
Graphics:
  Device-1: AMD Renoir vendor: Acer Incorporated ALI driver: amdgpu 
  v: kernel bus ID: 03:00.0 
  Device-2: Quanta HD User Facing type: USB driver: uvcvideo bus ID: 1-3:2 
  Display: x11 server: X.Org 1.20.8 driver: amdgpu FAILED: ati 
  unloaded: modesetting resolution: 1920x1080~60Hz 
  OpenGL: renderer: AMD RENOIR (DRM 3.38.0 5.8.6-1-MANJARO LLVM 10.0.1) 
  v: 4.6 Mesa 20.1.7 direct render: Yes 
Audio:
  Device-1: AMD vendor: Acer Incorporated ALI driver: snd_hda_intel 
  v: kernel bus ID: 03:00.1 
  Device-2: AMD Raven/Raven2/FireFlight/Renoir Audio Processor 
  vendor: Acer Incorporated ALI driver: snd_rn_pci_acp3x v: kernel 
  bus ID: 03:00.5 
  Device-3: AMD Family 17h HD Audio vendor: Acer Incorporated ALI 
  driver: snd_hda_intel v: kernel bus ID: 03:00.6 
  Sound Server: ALSA v: k5.8.6-1-MANJARO 
Network:
  Device-1: Intel Wi-Fi 6 AX200 driver: iwlwifi v: kernel bus ID: 01:00.0 
  IF: wlp1s0 state: up mac: <filter> 
Drives:
  Local Storage: total: 476.94 GiB used: 362.90 GiB (76.1%) 
  ID-1: /dev/nvme0n1 vendor: SK Hynix model: HFM512GDJTNI-82A0A 
  size: 476.94 GiB 
Partition:
  ID-1: / size: 180.69 GiB used: 97.39 GiB (53.9%) fs: ext4 
  dev: /dev/nvme0n1p7 
Swap:
  ID-1: swap-1 type: partition size: 4.00 GiB used: 0 KiB (0.0%) 
  dev: /dev/nvme0n1p6 
Sensors:
  System Temperatures: cpu: 39.4 C mobo: N/A gpu: amdgpu temp: 38 C 
  Fan Speeds (RPM): N/A 
Info:
  Processes: 275 Uptime: 2h 05m Memory: 7.21 GiB used: 3.06 GiB (42.4%) 
  Init: systemd Compilers: gcc: 10.2.0 clang: 10.0.1 Packages: 1610 
  Shell: Bash v: 5.0.18 inxi: 3.1.05

**Comment by fvulich** [comment]
Could you try with a different package? because depending on your Linux distribution some are better than the others. You can download them from [HERE](https://github.com/ramboxapp/community-edition/releases/latest) or AUR from [HERE](https://aur.archlinux.org/packages/rambox-bin/)

**Comment by sujay1844** [comment]
i use the package from the aur

**Comment by fvulich** [comment]
Could you try with a different package?

**Comment by sujay1844** [comment]
I tried the appimage, aur package and snap. Same issue

**Comment by marcoesposito1988** [comment]
Are you using KDE Manjaro? I have the same issue there. The window seems to be there, but it doesn't react to mouse events.

It seems to be an issue of the window manager: the program actually exits, but the window lies around. Meanwhile I found a workaround when this happens: to disable/reenable kwin OpenGL detection (this restarts the composer) with the shortcut `Alt + F12`.

I doubt that this is a Rambox issue, although it does happen to me only with this program (so far).

**Comment by bxabi** [comment]
Yes I have the same issue on KDE, only with rambox. I click the X, rambox gets minimized, but it stays embedded in the desktop background. Other electron apps don't have this problem.
